src/System.Private.CoreLib/shared/System/Diagnostics/Debug.cs

```diff
@@ -73,7 +74,7 @@ public static void Unindent()
         }
 
         [System.Diagnostics.Conditional("DEBUG")]
-        public static void Print(string message)
+        public static void Print(string? message)
```

Comment:
The below overload that takes a `params object[] args` should actually be `params object?[] args`.